utils/dbhelper.js

```diff
@@ -14,7 +14,7 @@ var DBHelper = function () {
  * @param error Function to call on error
  */
 DBHelper.query = function (sql, parameters, done, error) {
-    if (process.env.USE_SSL) {
+    if (process.env.USE_SSL && process.env.USE_SSL.toLowerCase() !== 'false') {
```

Comment:
Odd, that was happening on my Mac - process.env.USE_SSL was resolved to a string "false", and that is auto-casted to bool: true

my process.env entry is: USE_SSL=false